### ISSUE TITLE
Change access points from absolute path to relative path in <ipaddress>:8428 main web page 

### ISSUE THREAD

**Issue by gntsyt2003** [VictoriaMetrics/VictoriaMetrics#1858]
**Describe the bug**
When we setup a http proxy for victoriametrics, the access points in main web page (<ipaddress>:8428) was not directed to right location due to it uses absolute path like below,  if it can be changed to relative path, it is perfect. (By the way, vmalert main web page <ipaddress>:8880 using the relative paths already)

/vmui - Web UI
/targets - discovered targets list
/api/v1/targets - advanced information about discovered targets in JSON format
/metrics - available service metrics
/api/v1/status/tsdb - tsdb status page
/api/v1/status/top_queries - top queries
/api/v1/status/active_queries - active queries

**To Reproduce**
Setup a http proxy before victoriametrics, the access points could not direct to right location.

**Expected behavior**
Change absolute path to relative path in main web page(:8428), that will make the access points directed to right location no matter w/wo proxy. 

**Logs**
Check if any warnings or errors were logged by VictoriaMetrics components
or components in communication with VictoriaMetrics (e.g. Prometheus, Grafana).

**Screenshots**
![image](https://user-images.githubusercontent.com/30012907/143371364-9a8c23dd-41fe-404a-885e-8c60171d0333.png)

See how to setup monitoring here:
* [monitoring for single-node VictoriaMetrics](https://docs.victoriametrics.com/#monitoring)
* [montioring for VictoriaMetrics cluster](https://docs.victoriametrics.com/Cluster-VictoriaMetrics.html#monitoring)

**Version**
The line returned when passing `--version` command line flag to binary. For example:
```
$ ./victoria-metrics-prod --version
victoria-metrics-20211022-171155-tags-v1.68.0-0-gf64f62692
```

**Used command-line flags**
Please provide applied command-line flags used for running VictoriaMetrics and its components.

**Comment by YuriKravetc** [comment]
Hi,

You can try using the following flag:
```
 -http.pathPrefix string
    	An optional prefix to add to all the paths handled by http server. For example, if '-http.pathPrefix=/foo/bar' is set, then all the http requests will be handled on '/foo/bar/*' paths. This may be useful for proxied requests. See https://www.robustperception.io/using-external-urls-and-proxies-with-prometheus
```

**Comment by valyala** [comment]
VictoriaMetrics should use relative paths for ulrs at main page after the commit 90c542af122622bdf2104c98dc03f1e633df3c8d . This commit will be included in the next release.

**Comment by hacker** [comment]
Would that mean that `http.pathPrefix` becomes unnecessary? Because I recently had problem with path prefix  and helm chart — https://github.com/VictoriaMetrics/helm-charts/issues/261 —  though if pathPrefix is not necessary anymore that's not much of a problem.

**Comment by valyala** [comment]
> Would that mean that http.pathPrefix becomes unnecessary? Because I recently had problem with path prefix and helm chart — VictoriaMetrics/helm-charts#261 — though if pathPrefix is not necessary anymore that's not much of a problem.

The `-http.pathPrefix` command-line option is necessary if http proxy doesn't strip the prefix from url before sending the request to VictoriaMetrics. For example, if http proxy serves VictoriaMetrics requests at `/foo/bar` url prefix, then all the VictoriaMetrics components expect that all the requests contain the provided prefix.

Closing this feature request, since it is available starting from [v1.70.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases).

**Comment by hacker** [comment]
Ah, thanks, I didn't consider an option of reverse proxy not being able to manipulate path.
